src/solutions/solution_interface.jl

```diff
@@ -73,7 +73,11 @@ Base.@propagate_inbounds function Base.getindex(A::AbstractTimeseriesSolution, s
         if sym isa AbstractArray
             return A[collect(sym)]
         end
-        i = sym_to_index(sym, A)
+        if hasfield(typeof(A), :sym_map) && !isnothing(A.sym_map)
+            i = get(A.sym_map, sym, nothing)
+        else
+            i = sym_to_index(sym, A)
+        end
     elseif all(issymbollike, sym)
         if has_sys(A.prob.f) && all(Base.Fix1(is_param_sym, A.prob.f.sys), sym) ||
```

Comment:
`i` isn't defined in this branch. Maybe this causes the test failure.

Reply:
that branch always returns